test/uk/gov/hmcts/contino/YarnBuilderTest.groovy

```diff
@@ -59,9 +59,6 @@ class YarnBuilderTest extends Specification {
     when:
     builder.test()
     then:
-    1 * steps.sh(['script':'yarn check 1> /dev/null 2> /dev/null', 'returnStatus':true])
```

Comment:
This test was testing a bunch of implementation detail.

I've cleaned it up by removing the non interesting bits, and I've left this check in one place for the first test